packages/compute-baseline/README.md

```diff
@@ -0,0 +1,57 @@
+# [`compute-baseline`](https://github.com/web-platform-dx/web-features/)
+
+By the [W3C WebDX Community Group](https://www.w3.org/community/webdx/) and contributors.
+
+`compute-baseline` computes preliminary [Baseline statuses](https://github.com/web-platform-dx/web-features/blob/main/docs/baseline.md) from `@mdn/browser-compat-data` feature keys.
+You can use `compute-baseline` to help you find interopable web platform features, propose new [`web-features`](https://github.com/web-platform-dx/web-features/) features, or compare support between features.
+
+`compute-baseline` also provides utility classes for working with `@mdn/browser-compat-data` generally.
+
+## Limitations
+
+If you need authoritative Baseline statuses, check out the [`web-features`](https://github.com/web-platform-dx/web-features/tree/main/packages/web-features) package instead.
+
+Don't use `compute-baseline` to generate publishable Baseline statuses for arbitrary web platform features.
+The results of `compute-baseline` invocations have _not_ received editorial review.
+Strictly speaking, a Baseline status requires editorial review, so statuses generated by `compute-baseline` are tentative.
+If you need authoritative Baseline statuses, check out the [`web-features`](https://github.com/web-platform-dx/web-features/tree/main/packages/web-features) package instead.
+
+You can use `compute-baseline` to explore possibilities or do error correction.
+For example, you might use `compute-baseline` to hide a Baseline status, when showing a broader feature's status might be misleading.
+
+If you're not sure whether your application fits with the definition of Baseline, please [file an issue](https://github.com/web-platform-dx/web-features/issues/new).
+
+## Prerequisites
+
+To use this package, you'll need:
+
+- Node.js (a supported [current, active LTS, or maintenance LTS release](https://nodejs.org/en/about/previous-releases))
+
+## Install
+
+To install the package, run:
+
+`npm install --save compute-baseline`
+
+If you wish to specify which version of `@mdn/browser-compat-data` (or manage its upgrades explicitly, such as with Dependabot), then install the latest `@mdn/browser-compat-data` too.
+Run:
+
+`npm install --save @mdn/browser-compat-data@latest`
+
+## Usage
+
+```javascript
+import { computeBaseline } from "compute-baseline";
+
+computeBaseline({
+  compatKeys: ["css.properties.border-color"],
+  checkAncestors: false,
```

Comment:
Is it necessary to specify `checkAncestors` if it's false?

And example of bring-your-own-BCD would be helpful here.

Reply:
`withAncestors` is required. I put in https://github.com/web-platform-dx/web-features/pull/936 to remove this requirement. I'll push a change shortly illustrating the bring-your-own-BCD stuff.